engines/director/detection_tables.h

```diff
@@ -1930,7 +2017,8 @@ static const DirectorGameDescription gameDescriptions[] = {
 	MACGAME1_l("refixion3", "", "The Reindeer Story", "f5277c53bacd27936158dd3867e587e2", 392473, Common::JA_JPN, 300),
 
 	// Original Mac version is Director. Later DOS port uses MADE engine (also in ScummVM).
-	MACDEMO1("rodneyfs", "Demo", "Rodney's Funscreen demo", "9f0bb7ec7720e4f680ee3aa3d22c1c9d", 356866, 300),
+	MACDEMO1("rodneyfs", "Demo", "Rodney's Funscreen demo", "9f0bb7ec7720e4f680ee3aa3d22c1c9d", 356610, 300),
+	// MACDEMO1("rodneyfs", "Demo", "Sample Application",	"9f0bb7ec7720e4f680ee3aa3d22c1c9d", 356610, 300),
```

Comment:
Why is this one commented?

Reply:
Similarly to the playroom demo above, the genericness of the filename and it being identical to the existing demo is why I commented it